index.js

```diff
@@ -13,133 +13,167 @@
  * babel cli:
  *
  * babel [your-source-dir] --plugins=glamorous-to-emotion --presets=react,etc... --out-dir=[your-source-dir]
- * 
+ *
  * A demo can be seen at:
  * https://astexplorer.net/#/gist/7bc4771564a12c9f93c4904b3934aa1c/latests
  */
 
-module.exports = function (babel) {
-    const { types: t } = babel;
+module.exports = function(babel) {
+  const {types: t} = babel;
+
+  const fixContentProp = args => {
+    return args.map(argument => {
+      if (argument.type !== "ObjectExpression") {
+        return argument;
+      }
+      return {
+        ...argument,
+        properties: argument.properties.map(prop => {
+          if (prop.key.name !== "content") {
+            return prop;
+          }
 
-    return {
-        visitor: {
-            /**
-             * Find all JSX elements that use the old
-             * <glamorous.Something> notation and rewrite them to be
-             * plain ole somethings.
-             */
-            JSXElement(path) {
-                path.traverse({
-                    "JSXOpeningElement|JSXClosingElement"(path) {
-                        if (!t.isJSXMemberExpression(path.node.name)) {
-                            return;
-                        }
-                        if (path.node.name.object.name !== "glamorous") {
-                            return;
-                        }
-                        path.node.name = t.identifier(
-                            path.node.name.property.name.toLowerCase()
-                        );
-                    }
-                });
-            },
+          // Add quotes to the content property.
+          return {
+            ...prop,
+            value: t.stringLiteral(`"${prop.value.value}"`),
+          };
+        }),
+      };
+    });
+  };
 
-            /**
-             * Find glamorous import statements, including ThemeProvider
-             * imports and rewrite them to be emotion imports.
-             */
-            ImportDeclaration(path, { opts }) {
-                if (path.node.source.value !== "glamorous") {
-                    return;
-                }
+  const transformJSXAttributes = attrs => {
+    if (!attrs) return [];
+    const appendToCss = [];
+    let cssAttr = null;
+    const newAttrs = attrs.filter(attr => {
+      const {value, name, type} = attr;
+      if (type === "JSXSpreadAttribute") return true;
+      if (name.name === "css") {
```

Comment:
I think we can do better than `name.name` here in order to allow developers not as experienced with this stuff to understand what's happening and more clearly describe our intent.

Reply:
I didn't come up with `attr.name.name`. That's part of the AST API.

Reply:
I know that, but I'm suggesting aliasing the destructured name on line 51 in order to provide an identifier that is a little more understandable.